crates/bevy_render/src/render_resource/resource_macros.rs

```diff
@@ -0,0 +1,102 @@
+#[cfg(debug_assertions)]
+#[macro_export]
+macro_rules! render_resource_wrapper {
```

Comment:
I think its worth adding some documentation for the rationale behind this wrapper (and the criteria for removing it in the future?)